vm/systemSmartContracts/esdt.go

```diff
@@ -217,6 +227,67 @@ func (e *esdt) init(_ *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 	return vmcommon.Ok
 }
 
+func (e *esdt) initDelegationESDTOnMeta(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
```

Comment:
unit tests

Reply:
we have in systemSCs.go - but will come in next PRs